aws/rust-runtime/aws-config/test-data/profile-parser-tests.json

```diff
@@ -748,12 +850,48 @@
         "configFile": "[profile foo]\nname = value\n[profile in valid]\nx = 1\n[profile bar]\nname = value2"
       },
       "output": {
-        "profiles": {
-          "bar": {
-            "name": "value2"
+        "config": {
+          "profiles": {
+            "bar": {
+              "name": "value2"
+            },
+            "foo": {
+              "name": "value"
+            }
+          }
+        }
+      }
+    },
+    {
+      "name": "sso-session with properties",
+      "input": {
+        "configFile": "[sso-session foo]\nname = value\n[profile bar]"
+      },
+      "output": {
+        "config": {
+          "profiles": {
+            "bar": {}
+          },
+          "sso_sessions": {
+            "foo": {
+              "name": "value"
+            }
+          }
+        }
+      }
+    },
+    {
+      "name": "sso-session without the space before the name is a profile",
+      "input": {
+        "credentialsFile": "[sso-sessionfoo]\nname = value\n[bar]"
+      },
+      "output": {
+        "config": {
+          "profiles": {
+            "sso-sessionfoo": { "name": "value" },
+            "bar": {}
           },
-          "foo": {
-            "name": "value"
+          "sso_sessions": {
```

Comment:
should probably include a test of something like `sso-sesion blah` (a typo'd sso session key)

Reply:
Added in 3ab8cfac3748f7e42e1707abe80e5425ffa0ba95